src/main/java/io/pravega/connectors/hadoop/PravegaInputFormat.java

```diff
@@ -77,15 +92,24 @@ protected PravegaInputFormat(ClientFactory externalClientFactory) {
         final String deserializerClassName = Optional.ofNullable(conf.get(PravegaInputFormat.DESERIALIZER)).orElseThrow(() ->
```

Comment:
This variable is not being used..

Reply:
this var is being used by PravegaInputRecordReader.java. Put all user visible micros in a single file.